tests/topotests/bgp_default_originate/test_bgp_default_originate_topo1_3.py

```diff
@@ -890,57 +893,126 @@ def test_verify_default_originate_after_BGP_and_FRR_restart_p2(request):
             tc_name, result
         )
 
-    step(" BGP Daemon restart operation")
+    # Allow for verification of the table version
+    # as that restarting bgp on one router will cause
+    # r2's version number should go up as that r1
+    # is not directly connected to r2
+    # where?  Fuck if I know.
```

Comment:
We probably should "fix" this before merging :joy:

Reply:
fixed. but I disagree with the assessment